pkg/vere/io/ames.c

```diff
@@ -5,7 +5,7 @@
 #include "noun.h"
 #include "ur.h"
 
-#define FINE_PAGE       512             //  packets per page
+#define FINE_PAGE      4096             //  packets per page
 #define FINE_FRAG      1024             //  bytes per fragment packet
```

Comment:
What is a Fine page? Is it the maximal file size? Just trying to figure out to which change described in Arvo 6669 this corresponds.

Reply:
It's the max number of packets we read out in a scry result. Eg there's a 15MB file in our clay history. Before, that would require about 30 scry requests into arvo, now it requires 4, and each one loads up to 4096 entries into the cache.